pkg/sif/index.go

```diff
@@ -17,22 +17,24 @@ import (
 
 var _ v1.ImageIndex = (*imageIndex)(nil)
 
-// ImageIndexFromFileImage returns a v1.ImageIndex corresponding to f.
+// ImageIndexFromFileImage is a convenience function which constructs an
+// OCIFileImage from a sif.FileImage, and returns its RootIndex as
+// a v1.ImageIndex.
```

Comment:
Do you think we should mark this as deprecated, as with `Update`? If we want to move away from consumers working directly with the root index, might be counter productive to have this helper available in the long term?

```suggestion
// ImageIndexFromFileImage is a convenience function which constructs an
// OCIFileImage from a sif.FileImage, and returns its RootIndex as
// a v1.ImageIndex.
//
// Deprecated: Use OCIFileImage.RootIndex instead. ImageIndexFromFileImage will be removed in a future version.
```